# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../jakarta-regexp/remove-jakarta-site2.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- jakarta-regexp-1.3/build.xml	2003-09-02 20:56:46.000000000 +0200
+++ jakarta-regexp-1.3-patch/build.xml	2005-07-24 20:30:47.891346560 +0200
@@ -36,7 +36,7 @@
     <property name="optimize" value="on"/>
     <property name="deprecation" value="off"/>
     <property name="RETest.txt" value="docs/RETest.txt"/>
-    <property name="jakarta-site2.dir" value="../jakarta-site2"/>
+    <property name="jakarta-site2.dir" value="."/>
 
     <!-- Anakia -->
     <property name="docs.src" value="xdocs"/>
@@ -44,11 +44,8 @@
 
     <!-- Build classpath -->
     <path id="classpath">
-        <fileset dir="${jakarta-site2.dir}/lib">
-            <exclude name="ant*.jar"/>
-            <exclude name="maven*.jar"/>
-            <exclude name="*dvsl*.jar"/>
-            <include name="*.jar"/>
+        <fileset dir="${jakarta-site2.dir}">
+            <exclude name="*.jar"/>
         </fileset>
     </path>
 
